bundle/nxiq-operator-certified.clusterserviceversion.yaml

```diff
@@ -96,7 +96,7 @@ spec:
   customresourcedefinitions:
     owned:
     - kind: NexusIQ
-      name: nexusiqs.apm.nxiq.com
+      name: nexusiqs.sonatype.com
```

Comment:
where does the 's' come from in 'nexusiqs'?

Reply:
Red Hat's version of the operator had set the example of this referring to the plural name, and it passes validation/works, so that's what I kept. I can look into it again. I'm getting more confident in knowing when it's working or not.

Reply:
If you search through these files, the "plural" shows up multiple times, even as explicit configuration, but still, i can confirm it's use in each place.

Reply:
https://github.com/sonatype/operator-iq/search?q=nexusiqs&unscoped_q=nexusiqs

It does still stand out though, doesn't it. I look for further confirmation beyond what red hat provided. https://github.com/sonatype/operator-nxrm3/blob/original-from-redhat/bundle/sonatypenxrm-operator.v0.0.0.clusterserviceversion.yaml#L135

Reply:
The "name" to which it is referring comes from the full name of the CRD, and that is generated as a plural.
doc: https://docs.openshift.com/container-platform/4.1/applications/operator_sdk/osdk-generating-csvs.html#osdk-crds-owned_osdk-generating-csvs
our CRD here: https://github.com/sonatype/operator-iq/pull/3/files#diff-3bbed2728f60400faa6622125f3bce97R4
our CRD as generated initially by tooling: https://github.com/sonatype/operator-iq/commit/a73e28e29eed196d21214dd08566255306d80648#diff-52364aaf65e5b6f26b37d67352ed53a8R4

All are the plural version. Thanks for giving me something to learn. :)